randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1341,8 +1343,23 @@ Extra - room_id: [35]
   * Layers: default
   * Tunnel to Diffusion Connection Access/Other to Powamp Shaft
   * Extra - door_idx: (112,)
+  > In Narrow Tunnel
+      Missiles
+
+> In Narrow Tunnel; Heals? False
+  * Layers: default
   > Door to Owtch-Couch
+      Morph Ball
+  > Door to Aquarium Speedway
       Missiles and Morph Ball
+  > Other to Diffusion Connection Access
+      All of the following:
+          # Get into tunnel
+          Mid-Air Morph (Intermediate) or Can Bounce in Ball
+          Any of the following:
+              # Destroying the Missile Blocks
```

Comment:
This needs Diffusion missiles && missiles. Becaue missile is the ammo itself
Same with upper door -> insinde tunnel

Reply:
Yup, fixed for both paths